tests/test_all.py

```diff
@@ -310,6 +309,10 @@ def test_no_orphaned_packages(container_per_test: ContainerData) -> None:
         "sles-release",
         "ALP-dummy-release",
     }
+    # bsc#1219115
+    if OS_VERSION == "15.6":
+        known_orphaned_packages.union({'java-11-openjdk', 'java-11-openjdk-headless'})
```

Comment:
`union` does not modify in-place
```suggestion
        known_orphaned_packages = known_orphaned_packages.union({'java-11-openjdk', 'java-11-openjdk-headless'})
```